dptb/tests/test_to_pardiso.py

```
@@ -0,0 +1,229 @@
+        data = json.load(f)
+        
+    # Check Schema
+    assert "structure" in data, "JSON missing 'structure' key"
+    assert "basis_info" in data, "JSON missing 'basis_info' key"
+    
+    # Check Structure Data
+    assert data["structure"]["nsites"] == len(tbsys.atoms), "Atom count mismatch in JSON"
+    assert np.allclose(data["structure"]["positions"], tbsys.atoms.get_positions()), "Positions mismatch in JSON"
+    
+    # Check Basis Data
+    total_orbitals = data["basis_info"]["total_orbitals"]
```

Ruff (0.14.13)

Local variable `total_orbitals` is assigned to but never used

Remove assignment to unused variable `total_orbitals`

(F841)